packages/fast-form-generator-react/src/form/form-item.children.tsx

```diff
@@ -12,6 +12,7 @@ import styles from "./form-item.children.style";
 import { IFormItemChildrenClassNameContract } from "../class-name-contracts/";
 import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
 import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import { isDescendant } from "@microsoft/fast-web-utilities";
```

Comment:
Same comment here on isDescendant. I'm surprised you have access to this since I don't see it in the actual package.

Reply:
I added it to the package